src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -260,5 +277,41 @@ private static void UpdatePropertyForLocaleManifests(string propertyName, string
                 manifest.GetType().GetProperty(propertyName).SetValue(manifest, value);
             }
         }
+
+        /// <summary>
+        /// Compares the hashes of the original and updated manifests and returns true if the new manifest has an updated SHA256 hash.
+        /// </summary>
+        /// <param name="oldManifests">The original manifest object model.</param>
+        /// <param name="newManifests">The updated manifest object model.</param>
+        /// <returns>A boolean value indicating whether the updated manifest has new changes compared to the original manifest.</returns>
+        private bool CheckForUpdatedInstallerHash(Manifests oldManifests, Manifests newManifests)
+        {
+            InstallerManifest oldInstallerManifest = oldManifests.InstallerManifest;
+            List<string> hashes = new List<string>();
+
+            foreach (WingetCreateCore.Models.Installer.Installer installer in oldInstallerManifest.Installers)
+            {
+                if (!hashes.Contains(installer.InstallerSha256))
+                {
+                    hashes.Add(installer.InstallerSha256);
+                }
+            }
+
+            InstallerManifest newInstallerManifest = newManifests.InstallerManifest;
+
+            foreach (WingetCreateCore.Models.Installer.Installer installer in newInstallerManifest.Installers)
+            {
+                if (hashes.Contains(installer.InstallerSha256))
+                {
+                    continue;
+                }
+                else
+                {
+                    return true;
+                }
+            }
+
+            return false;

```

Comment:
Learn to love LINQ :)

```
            var oldHashes = oldInstallerManifest.Installers.Select(i => i.InstallerSha256).Distinct();
            var newHashes = newInstallerManifest.Installers.Select(i => i.InstallerSha256).Distinct();
            return newHashes.Except(oldHashes).Any();

Reply:
truly magic :) Thanks for the suggestion.